posts/seurat-cosmx-basics/index.qmd

```diff
@@ -0,0 +1,244 @@
+---
+title: "Introduction to visualizing CosMx data in Seurat"
+author:
+  - name: Claire Williams
+    affiliations:
+      - ref: nstg
+      - ref: crwilliams11
+affiliations:
+  - id: nstg
+    name: NanoString Technologies, Inc.
+    city: Seattle
+    state: WA
+  - id: crwilliams11
+    name: "Github: [@crwilliams11](https://github.com/crwilliams11)"
+toc: true
+toc-title: Contents
+toc-depth: 3
+toc-expand: 2
+toc-location: left
+number-sections: true
+number-depth: 4
+date: "2024-05-10"
+date-modified: last-modified
+categories: [visualization]
+draft: false
+---
+
+# Introduction 
+
+This is a non-comprehensive vignette to introduce the use of the Seurat package to visualize and work with data generated from the CosMx&#8482; Spatial Molecular Imager (SMI). This vignette does not cover analysis of data in Seurat but rather tries to address frequently asked questions we've received from customers on getting started with their data in Seurat. As additional resources, please see vignettes on the Seurat website: [Spatial Vignette](https://satijalab.org/seurat/articles/seurat5_spatial_vignette_2.html#human-lung-nanostring-cosmx-spatial-molecular-imager) and
+[Clustering Tutorial](https://satijalab.org/seurat/articles/pbmc3k_tutorial). Also see additional blog posts on our [scratch space](https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/about.html).
+
+For this vignette, we use a Seurat object made from the mouse brain public data set and assembled in the structure used by the Technology Access Program [TAP](https://nanostring.com/products/cosmx-spatial-molecular-imager/technology-access-program/); similar outputs are available from the AtoMx&#8482; Spatial Informatics Portal (SIP). To download raw data for this dataset, go [here](https://nanostring.com/products/cosmx-spatial-molecular-imager/ffpe-dataset/cosmx-smi-mouse-brain-ffpe-dataset/). 
+
+Like other items in our [CosMx Analysis Scratch Space](https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/about.html){target="_blank"},
+the usual [caveats](https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/about.html){target="_blank"} and [license](https://nanostring-biostats.github.io/CosMx-Analysis-Scratch-Space/license.html){target="_blank"} applies.
+This post will show you how to:
+
+- @sec-loading Load needed libraries and example dataset
+- @sec-data-structure Find important data in Seurat object
+- @sec-spatial-plotting Plot CosMx data in space
+- @sec-dim-reduction-plots Visualize dimension reduction data 
+
+
+
+
+# Data Loading {#sec-loading}
+Here we start from a Seurat object containing CosMx data and analysis stored in an rds file. To start from raw data instead, see the Seurat Spatial Vignette above.
```

Comment:
I think we should point out that this only works in certain configurations (e.g., when you download the Seurat with polygons and when you are using Seurat v 5+).

Something like, 
If you are exporting from AtoMx (v1.3+), be sure to export the Seurat object with the polygon coordinates.

Reply:
Good callout. There is now a note in section 2: "Many of the below functions require that you are working with [Seurat v5](https://satijalab.org/seurat/) and may not work in earlier versions. Additionally, if you are exporting a Seurat object from AtoMx (v1.3+), be sure to export the Seurat object with polygon coordinates and transcripts included to access all of the functionality below." 
Addressed in commit 4f6bcce.